Translation/HistoryManager.php

```diff
@@ -41,11 +41,11 @@ public function __construct(Repository $repository)
     public function history(Request $request)
     {
         $content = $this->parseJsonContent($request);
-        $boolFilter = new Bool();
-        $boolFilter->addToBool(new TermFilter('key', $content['key']));
-        $boolFilter->addToBool(new TermFilter('domain', $content['domain']));
-        $boolFilter->addToBool(new TermFilter('locale', $content['locale']));
-        $sort = new Sort('created_at', Sort::ORDER_DESC);
+        $boolFilter = new BoolQuery();
```

Comment:
Much better would be to avoid BoolQuery. If you add a `TermFilter` to the search it will form a bool clause automatically. 

```php
$search = $this->repository->createSearch();
$search->addFilter(new TermFilter('key', $content['key']));
$search->addFilter(new TermFilter('domain', $content['domain']));
$search->addFilter(new TermFilter('locale', $content['locale']));
$search->addSort(new FieldSort('created_at', FieldSort::DESC));
```